apps/pie-microsite/src/assets/styles/components/_tokensTable.scss

```diff
@@ -92,6 +92,32 @@ $tokensTable-item-min-height: 128px;
     }
 }
 
+.c-tokensTable-example-radius-container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: f.$color-background-subtle;
+    min-height: $tokensTable-spacing-container-height;
+
+    @include f.media('>=wide') {
+        display: block;
+        background-color: transparent;
+    }
+}
+
+.c-tokensTable-example-radius {
+    width: 80px;
+    height: 80px;
+    margin-top: 16px;
```

Comment:
we can use `f.spacing(d)` here instead of `16px`

[reference](https://github.com/justeat/fozzie/blob/eb5775f47d30e21b4b60407da9fa5dd36a3ecbc9/src/scss/settings/_variables.scss#L98)